microsoft-edge/webview2/release-notes.md

```diff
@@ -46,7 +189,7 @@ For full API compatibility, this version of the WebView2 SDK requires WebView2 R
 
 ### General
 
-This WebView2 SDK release has the same bug fixes that are in WebView2 SDK 1.0.1466-prerelease. See **Bug fixes** in the following section.
+This WebView2 SDK release has the same bug fixes that are in WebView2 SDK 1.0.1414-prerelease. See **Bug fixes** in the following section.
 
 
 <!-- ====================================================================== -->

```

Comment:
Change from:
```
#### <a id="bug-fixes-1466-prerelease"/>Bug fixes
```
to:
```
#### <a id="bug-fixes-1466-prerelease"/>Bug fixes
```
proposed way to enable linking directly to here, per 
https://learn.microsoft.com/en-us/contribute/how-to-write-links#explicit-anchor-links

Reply:
fixed in local